.docker/scripts/init-ext.sh

```diff
@@ -31,7 +31,12 @@ install_requirements . dev-requirements requirements-dev
 for extension in . `ls -d $SRC_DIR/ckanext-*`; do
     install_requirements $extension requirements pip-requirements
 done
+
+make prepare
+make sync install py2=1 upgrade_requirements= use_2020_resolver= root_dir="$VENV_DIR/src"
```

Comment:
Why introduce `make` instead of just using `pip`?

Reply:
Please, contact Iaroslav about this one

Reply:
@iaroslav13 Can you comment on this?

Reply:
@mutantsan please drop these lines. Needs a line each in dev-requirements.txt